deps/windows-install.py

```diff
@@ -14,7 +14,6 @@
 env_dist = os.environ
 downloadBaseUrl = env_dist['NODE_RDKAFKA_NUGET_BASE_URL'] if 'NODE_RDKAFKA_NUGET_BASE_URL' in env_dist else 'https://globalcdn.nuget.org/packages/'
 librdkafkaNugetUrl = downloadBaseUrl + 'librdkafka.redist.{}.nupkg'.format(librdkafkaVersion)
-print 'download librdkafka form ' + librdkafkaNugetUrl
```

Comment:
I think we should better replace it with:
`print('download librdkafka form ' + librdkafkaNugetUrl)`